src/main/java/com/rocketden/main/service/ProblemService.java

```diff
@@ -92,10 +92,15 @@ public ProblemDto getProblem(String problemId) {
             throw new ApiException(ProblemError.NOT_FOUND);
```

Comment:
Not allowed to comment on it but thoughts on changing lines 78-82 to return a generic `ProblemError.EMPTY_FIELD` error if either setting is missing?

Reply:
Yeah I'm ok with that 👍